README.md

```diff
@@ -49,8 +49,19 @@ Contains the JSON schemas used to validate the sources in the integration tests.
 
 ## GTFS Realtime Data Structure
 
-This is still under discussion. [Please comment to share your thoughts](https://github.com/MobilityData/mobility-database-catalogs/issues/36). [You can view what is currently drafted in the working document](https://docs.google.com/document/d/1Mlz3AXHItInitsOEAKKi8hZV9d3t3gpFbyIZGm67ESU/edit#heading=h.i8ficnh0oxru).
-
+|Field Name|Type|Presence|Definition|                                                                     
```

Comment:
I know that this table structure is the one we used for the GTFS Schedule Data Structure, but does it represent the CSV, the JSON schema or another representation (the variables in the operations for instance)? 

I think the table structure if good for now, but if the goal is to represent the CSV or the JSON schema, we should think about adjusting it. For example, in the CSV, `direct_download_url` is `urls.direct_download`, and in the JSON schema, `urls` is an object in which there is the `direct_download` URL.